### ISSUE TITLE
Passing strings to functions

### ISSUE THREAD

**Issue by wcornwell** [richfitz/remake#58]
```
Error in .remake_add_targets_implied(obj) : 
  Implicitly created targets must all be files:
 - PET: (in pet.part.res) -- did you mean: pet.part.res
```

I'm just trying to pass a string "PET" to the function.  Works fine outside of remake.

Not sure if this is a bug of if I'm not using remake properly.

**Comment by richfitz** [comment]
Use the `I()` wrapper.  So it'd look like `your_function(I("PET"))`

**Comment by wcornwell** [comment]
works :fireworks:

**Comment by wcornwell** [comment]
you can close this if you want.  

do you want me to keep posting little detail things that could go into future documentation?

**Comment by richfitz** [comment]
If you've seen details to be aware of, please add them, sure.

**Comment by wcornwell** [comment]
I get this error a lot:
```
Error in target_build(target, obj$store, obj$verbose$quiet_target) : 
  Can't build implicit targets
```
Took me a while to work out that it means "Input file not found / path to input file incorrect".  Maybe a more informative error message for that one?  Since it will come up fairly frequently for users.

**Comment by richfitz** [comment]
Thanks will; I agree that the error messages could be more informative, but working out the context is the trick. Any chance you could give a minimal example or describe more fully how to trigger and I'll sort it out.

**Comment by wcornwell** [comment]
Reproducible example is hard, but I think that's the error message whenever a path is wrong for a file input.  

```
temp.map:
    command: read.cru.mean.12("grid_10min_tmp.dat")
```

If that file doesn't exist, or the path is wrong then I get the error.  *Exceedingly* minor issue, but it did have me scratching my head for a little while...

**Comment by richfitz** [comment]
Right.  There should be a warning earlier in the the output saying something like:

```
Warning "grid_10min_tmp.dat" not found.  Did you mean, etc, etc
```

right?

**Comment by wcornwell** [comment]
Yes. Exactly.

**Comment by wlandau** [comment]
It would be great if the use of `I()` were demonstrated in an example and/or explained in the docs or README.
